agent/app/agent.go

```diff
@@ -23,8 +23,9 @@ import (
 	"github.com/aws/amazon-ecs-agent/agent/logger"
 	"github.com/aws/amazon-ecs-agent/agent/logger/field"
 
-	"github.com/aws/amazon-ecs-agent/agent/doctor"
+	dockerdoctor "github.com/aws/amazon-ecs-agent/agent/doctor"
```

Comment:
The distinction between `dockerdoctor` and `doctor` might not be clear to someone who comes upon this code, can we just throw a comment in here?

Reply:
Good idea, will do.